### PR TITLE
docs(synthetics): canary schedule section in README

### PR DESCRIPTION
**Pull request by jonrhall**
This PR adds more extensive documentation for how to schedule CloudWatch Synthetics canary cron jobs via CDK, because today the service documentation for how to schedule cron jobs is lacking and takes developers a lot of investigation to figure out how to make it work successfully.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/aws-synthetics/README.md [hunk 1]
```diff
@@ -87,6 +87,22 @@ The Canary code will be executed in a lambda function created by Synthetics on y
 
 To learn more about Synthetics capabilities, check out the [docs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Canaries.html).
 
+
+### Canary Schedule
+You can set the canary `schedule` property by assigning it a value with the `synthetics.Schedule` [class](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-synthetics.Schedule.html).
+
+Configure a run rate of up to 60 minutes with `synthetics.Schedule.rate`:
+
+```js
```
Comment (BenChaimberg) [inline packages/@aws-cdk/aws-synthetics/README.md:96]:
```suggestion
```ts
```

File: packages/@aws-cdk/aws-synthetics/README.md [hunk 2]
```diff
@@ -87,6 +87,22 @@ The Canary code will be executed in a lambda function created by Synthetics on y
...
+synthetics.Schedule.rate(Duration.minutes(5)), // Run every 5 minutes.
+```
+
+If expanded access to scheduling the canary is needed, use `synthetics.Schedule.expression` with [cron expressions, as defined here](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Canaries_cron.html), for example:
+
+```js
```
Comment (BenChaimberg) [inline packages/@aws-cdk/aws-synthetics/README.md:102]:
```suggestion
```ts
```

File: packages/@aws-cdk/aws-synthetics/README.md [hunk 3]
```diff
@@ -87,6 +87,22 @@ The Canary code will be executed in a lambda function created by Synthetics on y
 
 To learn more about Synthetics capabilities, check out the [docs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Canaries.html).
 
+
+### Canary Schedule
+You can set the canary `schedule` property by assigning it a value with the `synthetics.Schedule` [class](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-synthetics.Schedule.html).
```
Comment (BenChaimberg) [inline packages/@aws-cdk/aws-synthetics/README.md:92]: "set the canary schedule property" is not really a helpful statement, how about something like "Specify the schedule on which a canary runs by providing a `Schedule` value to the `schedule` property."?

File: packages/@aws-cdk/aws-synthetics/README.md [hunk 4]
```diff
@@ -87,6 +87,22 @@ The Canary code will be executed in a lambda function created by Synthetics on y
...
+
+Configure a run rate of up to 60 minutes with `synthetics.Schedule.rate`:
+
+```js
+synthetics.Schedule.rate(Duration.minutes(5)), // Run every 5 minutes.
+```
+
+If expanded access to scheduling the canary is needed, use `synthetics.Schedule.expression` with [cron expressions, as defined here](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Canaries_cron.html), for example:
```
Comment (BenChaimberg) [inline packages/@aws-cdk/aws-synthetics/README.md:100]:
```suggestion
If expanded access to scheduling the canary is needed, use `Schedule.expression` with [cron expressions](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Canaries_cron.html):
```

File: packages/@aws-cdk/aws-synthetics/README.md [hunk 5]
```diff
@@ -87,6 +87,22 @@ The Canary code will be executed in a lambda function created by Synthetics on y
...
+
+```js
+synthetics.Schedule.expression('cron(0 0,8,16 * * ? *)'), // Run at 12am, 8am, 4pm UTC every day
+```
+
```
Comment (BenChaimberg) [inline packages/@aws-cdk/aws-synthetics/README.md:105]: please also add a small section about `once`

File: packages/@aws-cdk/aws-synthetics/README.md [hunk 6]
```diff
@@ -87,6 +87,22 @@ The Canary code will be executed in a lambda function created by Synthetics on y
...
 To learn more about Synthetics capabilities, check out the [docs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Canaries.html).
 
+
+### Canary Schedule
+You can set the canary `schedule` property by assigning it a value with the `synthetics.Schedule` [class](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-synthetics.Schedule.html).
+
+Configure a run rate of up to 60 minutes with `synthetics.Schedule.rate`:
+
+```js
+synthetics.Schedule.rate(Duration.minutes(5)), // Run every 5 minutes.
```
Comment (BenChaimberg) [inline packages/@aws-cdk/aws-synthetics/README.md:97]:
```suggestion
Schedule.rate(Duration.minutes(5)), // Run every 5 minutes.
```

File: packages/@aws-cdk/aws-synthetics/README.md [hunk 7]
```diff
@@ -87,6 +87,22 @@ The Canary code will be executed in a lambda function created by Synthetics on y
 
 To learn more about Synthetics capabilities, check out the [docs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Canaries.html).
 
+
+### Canary Schedule
+You can set the canary `schedule` property by assigning it a value with the `synthetics.Schedule` [class](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-synthetics.Schedule.html).
+
+Configure a run rate of up to 60 minutes with `synthetics.Schedule.rate`:
```
Comment (BenChaimberg) [inline packages/@aws-cdk/aws-synthetics/README.md:94]:
```suggestion
Configure a run rate of up to 60 minutes with `Schedule.rate`:
```

File: packages/@aws-cdk/aws-synthetics/README.md [hunk 8]
```diff
@@ -87,6 +87,22 @@ The Canary code will be executed in a lambda function created by Synthetics on y
...
+
+```js
+synthetics.Schedule.rate(Duration.minutes(5)), // Run every 5 minutes.
+```
+
+If expanded access to scheduling the canary is needed, use `synthetics.Schedule.expression` with [cron expressions, as defined here](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Canaries_cron.html), for example:
+
+```js
+synthetics.Schedule.expression('cron(0 0,8,16 * * ? *)'), // Run at 12am, 8am, 4pm UTC every day
```
Comment (BenChaimberg) [inline packages/@aws-cdk/aws-synthetics/README.md:103]:
```suggestion
Schedule.expression('cron(0 0,8,16 * * ? *)'), // Run at 12am, 8am, 4pm UTC every day
```

**Review by BenChaimberg — CHANGES_REQUESTED** [review]
thank you for improving our documentation! these are the most important contributions in my book, so sorry for the massive comment-to-changed-line ratio

**Review by kaizen3031593 — APPROVED** [review]

**Review by kaizen3031593 — APPROVED** [review]

**Pull request merged**